test/quick_test/test_stats.py

```diff
@@ -0,0 +1,267 @@
+from .conftest import *
+
+from pathlib import Path
+
+import pandas as pd
+import pytest
+import yaml
+
+from .common import *
+
+from logging import getLogger
+
+logger = getLogger(__name__)
+
+
+@pytest.fixture
+def thresholds():
+    """
+    Load the thresholds from the given file path.
+
+    Returns
+    -------
+    default_threshold : float
+        Default threshold value.
+    thresholds : dict
+        Dictionary containing the thresholds
+    """
+
+    thresholds_file = Path(__file__).parent / "data/thresholds/aseg.stats.yaml"
+
+    # Open the file_path and read the thresholds into a dictionary
+    with open(thresholds_file, "r") as file:
+        data = yaml.safe_load(file)
+        default_threshold = data.get("default_threshold")
+        thresholds = data.get("thresholds", {})
+
+    return default_threshold, thresholds
+
+
+def load_stats_file(test_subject: Path):
+    """
+    Load the stats file from the given file path.
+
+    Parameters
+    ----------
+    test_subject : Path
+        Path to the test subject.
+
+    Returns
+    -------
+    stats_file : Path
+    """
+
+    files = os.listdir(test_subject / "stats")
+
+    if "aseg.stats" in files:
+        return test_subject / "stats" / "aseg.stats"
```

Comment:
in the end we want this test to check both of these files

Reply:
I would like to get the runner working on the server before making this change